### ISSUE TITLE
[Plan for Linux users] - ip-api.com blocked by Pihole

### ISSUE THREAD

**Issue by poelarends** [AmirrezaJaberi/FIREAC#77]
Just a tip, but your AC blocked me from connecting to my server. Turns out it was my pi-hole automatically blocking ip-api. Just making an issue here for anyone facing the same issue. :)
Thanks for all your work!

**Comment by AmirrezaJaberi** [comment]
Hi @poelarends,
Please tell me more about this bug

Why ip-api block you ? (for VPN or Hosting OR it's some problem in detect)

**Comment by poelarends** [comment]
Hi,

It's an automated block by my pi-hole. The domain was added to one of the
online blacklists. And without that domain, the loading screen hangs on
deferring connection when using FIREAC.
So I whitelisted the domain, and now the problem is gone. Seeing that this
is a weird issue, I thought I'd mention it to help anyone facing this same
issue.

Op wo 6 sep. 2023 22:05 schreef Amirreza Jaberi ***@***.***>:
